CMakeLists.txt

```diff
@@ -825,6 +825,12 @@ if(CAPSTONE_BUILD_SHARED_LIBS)
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     )
     target_compile_definitions(capstone PUBLIC CAPSTONE_SHARED)
+    # Build pdb file for dll on Windows
+    if(WIN32)
+        message("Enabling PDB for Windows")
+        set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /Zi")
+        set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /DEBUG /OPT:REF /OPT:ICF")
```

Comment:
Did you also try it with `set(CMAKE_MSVC_DEBUG_INFORMATION_FORMAT "ProgramDatabase")`?
It hopefully should save you specifying the linker flags below.

Reply:
`set(CMAKE_MSVC_DEBUG_INFORMATION_FORMAT "ProgramDatabase")` seems to have the same effect as line 831, but it won't generate .pdb file without line 832